share/scripts/inverse/iie_stat_incl_generic.sh

```diff
@@ -0,0 +1,68 @@
+#! /bin/bash
+#
+# Copyright 2009-2017 The VOTCA Development Team (http://www.votca.org)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+if [ "$1" = "--help" ]; then
+cat <<EOF
+${0##*/}, version %version%
```

Comment:
There is a lot of code duplication here, could you make this part of `calc_rdf_generic.sh` or is that too hard.
If not that is ok, too.